OpenSSL/test/test_crypto.py

```diff
@@ -2766,8 +2886,108 @@ def test_attribute(self):
         on an instance of :py:obj:`PKCS7Type`, :py:obj:`AttributeError` is
         raised.
         """
-        pkcs7 = load_pkcs7_data(FILETYPE_PEM, pkcs7Data)
-        self.assertRaises(AttributeError, getattr, pkcs7, "foo")
+        pkcs7 = load_pkcs7_data(FILETYPE_PEM, pkcs7signedData)
+        self.assertRaises(AttributeError, getattr, pkcs7, b"foo")
```

Comment:
this breaks on Python 3: https://travis-ci.org/pyca/pyopenssl/jobs/85629742

`getattr` takes native strings, not bytes.